src/strings/PowerFxResources.en-US.resx

```diff
@@ -2398,37 +2398,54 @@
     <value>Invalid json pointer.</value>
     <comment>Generic json pointer parsing error.</comment>
   </data>
+  <data name="None" xml:space="preserve">
+    <value>None</value>
+    <comment>Function category name</comment>
```

Comment:
>Function category name [](http://example.com/codeflow?start=13&length=22)

Suggestion for names and descriptions

- None: Function category name - function without pre-defined categories
- Text: Function category name - function that works on text values
- Logical: Function category name - function that works on Boolean values
- Table: Function category name - function that works on tabular data
- Behavior: Function category name - function that needs to be executed in behavior contexts (i.e., not purely functional, with side effects)
- Date and time: Function category name - function that works on date/time values
    - Please use "Date and ***time*** (sentence casing)
- Math and statistical: Function category name - functions for mathematical and/or statistical operations
    - Please use "Math and ***statistical*** (sentence casing)
- User-defined: Function category name - function that is defined by the user, not a built-in function
- Information: Function category name - function that returns information about the environment where it is running
- Color: Function category name - function that works on or returns color values
- Service: Function category name - function that comes from a web service, typically using REST (REpresentational State Transfer) communication
- Component: Function category name - function defined in components within the environment where it is running

Reply:
Great suggestions. Thanks.